src/edu/csus/ecs/pc2/ui/MultipleDataSetPane.java

```diff
@@ -478,7 +479,20 @@ protected void loadDataFiles() {
             dump(problemDataFiles, "debug before load");
         }
         try {
+
+            String sampleBaseDirectoryName = new File(baseDirectoryName).getParent() + File.separator +"sample";
+            boolean ishere = new File(sampleBaseDirectoryName).isDirectory();
```

Comment:
The value of the local variable `isHere` is never used and generates a warning in Eclipse.  If it's not going to be used, please remove it.

Reply:
will remove.